install/auto_installer.sh

```diff
@@ -0,0 +1,150 @@
+#!/bin/bash
+
+echo "======================================================"
+echo "Welcome to Mellivora CTF auto-installer"
+echo "Auto Installer works on any Ubuntu 16.04 server setup."
+echo "======================================================"
+read -n 1 -s -r -p "Press any key to start the installation."
+echo ""
+
+# Updating & Upgrading
+
+sudo apt-get update && sudo apt-get -y upgrade
+
+echo "======================================================"
+echo "Select **LAMP server** with space and then follow prompts."
+echo "======================================================"
+read -n 1 -s -r -p "Press any key to start"
+echo ""
+
+sudo apt-get -y install tasksel && sudo tasksel
```

Comment:
Unless you expressly want to use tasksel, this can actually probably be replaced with ``sudo apt-get -y install lamp-server^``